src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -122,8 +122,14 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
                                     </span>
                                 </div>)
                             }
-                            <LoadingIndicator isLoading={this.props.store.clinicalDataForSamples.isPending} />
-                            {!this.props.store.clinicalDataForSamples.isPending && (
+                            <LoadingIndicator
+                                isLoading={
+                                    this.props.store.clinicalDataForSamples.isPending ||
```

Comment:
we should probably hide this fallback thing.  Like this wrap this in another remoteData.  It's just too much to grock.